src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -257,57 +257,57 @@
     <form method="{{ form_method|lower }}" action="{{ action }}"{% for attrname, attrvalue in attr %} {{ attrname }}="{{ attrvalue }}"{% endfor %}{% if multipart %} enctype="multipart/form-data"{% endif %}>
     {%- if form_method != method -%}
         <input type="hidden" name="_method" value="{{ method }}" />
-    {%- endif %}
-{%- endblock form_start %}
+    {%- endif -%}
+{%- endblock form_start -%}
 
-{% block form_end -%}
-    {% if not render_rest is defined or render_rest -%}
+{%- block form_end -%}
+    {%- if not render_rest is defined or render_rest -%}
         {{ form_rest(form) }}
     {%- endif -%}
     </form>
-{%- endblock form_end %}
+{%- endblock form_end -%}
 
-{% block form_enctype -%}
+{%- block form_enctype -%}
     {% if multipart %}enctype="multipart/form-data"{% endif %}
-{%- endblock form_enctype %}
+{%- endblock form_enctype -%}
 
-{% block form_errors -%}
-    {% if errors|length > 0 -%}
+{%- block form_errors -%}
+    {%- if errors|length > 0 -%}
     <ul>
         {%- for error in errors -%}
             <li>{{ error.message }}</li>
         {%- endfor -%}
     </ul>
-    {%- endif %}
-{%- endblock form_errors %}
+    {%- endif -%}
+{%- endblock form_errors -%}
 
-{% block form_rest -%}
-    {% for child in form -%}
-        {% if not child.rendered -%}
+{%- block form_rest -%}
+    {%- for child in form -%}
+        {%- if not child.rendered -%}
             {{ form_row(child) }}
-        {%- endif %}
-    {%- endfor %}
-{%- endblock form_rest %}
+        {%- endif -%}
+    {%- endfor -%}
+{%- endblock form_rest -%}
 
 {# Support #}
 
-{% block form_rows -%}
-    {% for child in form -%}
+{%- block form_rows -%}
+    {%- for child in form -%}
         {{ form_row(child) }}
-    {%- endfor %}
-{%- endblock form_rows %}
+    {%- endfor -%}
+{%- endblock form_rows -%}
 
-{% block widget_attributes -%}
+{%- block widget_attributes -%}
     id="{{ id }}" name="{{ full_name }}"{% if read_only %} readonly="readonly"{% endif %}{% if disabled %} disabled="disabled"{% endif %}{% if required %} required="required"{% endif %}{% if max_length %} maxlength="{{ max_length }}"{% endif %}{% if pattern %} pattern="{{ pattern }}"{% endif %}
-    {%- for attrname, attrvalue in attr %} {% if attrname in ['placeholder', 'title'] %}{{ attrname }}="{{ attrvalue|trans({}, translation_domain) }}"{% else %}{{ attrname }}="{{ attrvalue }}"{% endif %}{% endfor %}
-{%- endblock widget_attributes %}
+    {%- for attrname, attrvalue in attr -%} {% if attrname in ['placeholder', 'title'] %}{{ attrname }}="{{ attrvalue|trans({}, translation_domain) }}"{% else %}{{ attrname }}="{{ attrvalue }}"{% endif %}{%- endfor -%}
```

Comment:
this change is wrong. The space between attributes must be kept, otherwise the HTML is invalid (see failing tests)